packages/sdk/src/types/ConnectorTypes.ts

```diff
@@ -35,13 +35,15 @@ export type QueryOptions = {
     sortOrder?: SortOrder | null;
 };
 
-export type ConnectorCapabilities = {
-    filtering?: boolean;
-    upload?: boolean;
-    query?: boolean;
-    remove?: boolean;
-    copy?: boolean;
-    detail?: boolean;
+export type FontConnectorCapabilities = {
+    query: boolean;
```

Comment:
same here, filtering should be kept, it is a part of the query function. It will allow us to show a search bar for a connector or not.